doc/classes/Projection.xml

```diff
@@ -131,152 +151,194 @@
 			</description>
 		</method>
 		<method name="get_pixels_per_meter" qualifiers="const">
 			<return type="int" />
 			<param index="0" name="for_pixel_width" type="int" />
 			<description>
+				Returns the number of pixels with the given pixel width displayed per meter, after this [Projection] is applied.
 			</description>
 		</method>
 		<method name="get_projection_plane" qualifiers="const">
 			<return type="Plane" />
 			<param index="0" name="plane" type="int" />
 			<description>
+				Returns the clipping plane of this [Projection] whose index is given by [param plane].
+				[param plane] should be equal to one of [constant PLANE_NEAR], [constant PLANE_FAR], [constant PLANE_LEFT], [constant PLANE_TOP], [constant PLANE_RIGHT], or [constant PLANE_BOTTOM].
 			</description>
 		</method>
 		<method name="get_viewport_half_extents" qualifiers="const">
 			<return type="Vector2" />
 			<description>
+				Returns the dimensions of the viewport plane that this [Projection] projects positions onto, divided by two.
 			</description>
 		</method>
 		<method name="get_z_far" qualifiers="const">
 			<return type="float" />
 			<description>
+				Returns the distance for this [Projection] beyond which positions are clipped.
 			</description>
 		</method>
 		<method name="get_z_near" qualifiers="const">
 			<return type="float" />
 			<description>
+				Returns the distance for this [Projection] before which positions are clipped.
 			</description>
 		</method>
 		<method name="inverse" qualifiers="const">
 			<return type="Projection" />
 			<description>
+				Returns a [Projection] that performs the inverse of this [Projection]'s projective transformation.
 			</description>
 		</method>
 		<method name="is_orthogonal" qualifiers="const">
 			<return type="bool" />
 			<description>
+				Returns [code]true[/code] if this [Projection] performs an orthogonal projection.
 			</description>
 		</method>
 		<method name="jitter_offseted" qualifiers="const">
 			<return type="Projection" />
 			<param index="0" name="offset" type="Vector2" />
 			<description>
+				Returns a [Projection] with the X and Y values from the given [Vector2] added to the first and second values of the final column respectively.
 			</description>
 		</method>
 		<method name="perspective_znear_adjusted" qualifiers="const">
 			<return type="Projection" />
 			<param index="0" name="new_znear" type="float" />
 			<description>
+				Returns a [Projection] with the near clipping distance adjusted to be [param new_znear].
+				[b]Note:[/b] The original [Projection] must be a perspective projection.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="w" type="Vector4" setter="" getter="" default="Vector4(0, 0, 0, 1)">
+			The fourth column of the [Projection].
 		</member>
 		<member name="x" type="Vector4" setter="" getter="" default="Vector4(1, 0, 0, 0)">
+			The first column of the [Projection].
```

Comment:
```suggestion
			The projection matrix's X vector (column 0). Equivalent to array index [code]0[/code].
```

I think we should keep it similar to other structures. I copied this text from Basis except replaced basis with projection. It's up for debate, I'm not sure if we should have `[Projection]` or not since we are already in the documentation for this type.